internal/common/utils/CommonFunctions.go

```diff
@@ -23,3 +25,17 @@ func GenerateRandomColour() Colour {
 func GenerateRandomFloat(min float64, max float64) float64 {
 	return min + rand.Float64()*(max-min)
 }
+
+// this function will take in a list of maps from ids to their corresponding vote (yes/ no in the case of acceptance)
+// and retunr a list of ids that can be accepted according to some metric (ie more than half voted yes)
+// ranked according to a metric (ie overall number of yes's)
+func GetAcceptanceRanking([]map[uuid.UUID]bool) []uuid.UUID {
```

Comment:
This seems a strange place to put GetAcceptanceRanking and WinnerFromDist. Maybe it's time to create a new file called VoteHandler.go where these functions live?

Reply:
Agreed. Since there's a physics engine, it might be nice to have a governance engine / voting engine too, say